vertical-pod-autoscaler/pkg/admission-controller/config.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2018 The Kubernetes Authors.
-
```

Comment:
Please don't modify the Copyright comment.

Reply:
that was by mistake